spacetemplate/assets/legacy.yaml

```diff
@@ -61,7 +71,10 @@ work_item_types:
   extends: "86af5178-9b41-469b-9096-57e5155c3f31"
   name: Scenario
   can_construct: yes
-  description: TBD
+  description: A scenario identifies a real-world user problem to resolve.
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.